### ISSUE TITLE
Better xsltparam handling

### ISSUE THREAD

**Issue by fsundermeyer** [openSUSE/daps#263]
Rather than using

```
--xsltparam "--xsltparam key=value --xsltparam key2=value2 --param key=value"
```

we should implement somethin more intuitive like

```
--xsltparam "key=value" --xsltparam "key2=value2" --param "key=value"
```

For the DC files it would be

XSLTPARAM="key=value key2=value2"
PARAM="key=value"

**Comment by tomschr** [comment]
I've created the [Gist `parse_param.sh`](https://gist.github.com/tomschr/59126140d7eca4c006ab) to make your live a little bit easier. :smile:

**Comment by tomschr** [comment]
See related issue #65

**Comment by fsundermeyer** [comment]
The command line part is solved with commit 343d992

**Comment by tomschr** [comment]
Wasn't closed automatically ("Fixes" should be followed by a hash character, followed by the issue number).
